Generate RAG Code

hrishipathak opened the pull request:
Added a recipe (python notebook) for Generating RAG Code.

rawkintrevo commented on recipes/Generate_RAG_Code/.gitkeep:
```diff
@@ -0,0 +1 @@
+
```
delete this file?

hrishipathak replied:
Deleted file .gitkeep

rawkintrevo reviewed (changes requested):
please rebase on main and `recipes/Generate_RAG_Code/Generate_RAG_Code.ipynb`
to the appropriate file in `.github/workflows`

Also to the same file in `.github/notebook_lists`.

rawkintrevo commented:
Thanks for the contribution @hrishipathak - delete the empty file (assuming it was created by accident) and clear your outputs, and add to the workflows per my last comment and I think you'll be good to merge

hrishipathak commented:
Hi Trevor @rawkintrevo 

Thanks for your review and comments. I've completed the following changes:-
1. Deleted the empty file (.gitkeep)
2. Notebook outputs cleared
3. Added `recipes/Generate_RAG_Code/Generate_RAG_Code.ipynb` to `.github/workflows/vanilla_workflow.yaml`
4. Added `recipes/Generate_RAG_Code/Generate_RAG_Code.ipynb` to `.github/notebook_lists/vanilla_notebooks.txt`
5. Modified the code as per latest `ibm-granite-community/granite-kitchen` package

Please review and let me know if any more changes are required. Thank you.
